toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -334,6 +340,46 @@ Must be a multiple of 1 MiB.
 
 The partitions to provision on the disk.
 
+## pxe type
+
+Specifies the PXE-specific configuration for the generated OS artifacts.
+
+<div id="pxe-isoimageurl"></div>
+
+### isoImageUrl [string]
```

Comment:
It might be better to have 2 mutually exclusive fields: `isoImageBaseUrl` (which adds ISO file name) and `isoImageFileUrl` (which doesn't add the ISO file name). That way, it avoids the behavior changing based on the format of the URL.

Reply:
okay, more structure is good.

Reply:
changed.

Reply:
added.